ci/do_ci.sh

```diff
@@ -85,11 +85,19 @@ CI_TARGET=$1
 
 if [[ $# -gt 1 ]]; then
   shift
+  # If this is a fuzz test, add _with_libfuzzer
+  if [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
```

Comment:
Do you need this two ifs here? Move them to below in `elif [[ "$CI_TARGET" == "bazel.fuzz" ]]; then` and ingest from $TEST_TARGETS?

Reply:
Ingested from $TEST_TARGET, the only awkward thing about it is that the user has to specify the "_with_libfuzzer" name for this to work.

Reply:
I guess that's fine, at least wildcards like `//test/server/...` works, (which didn't work with previous one), it is hard to make this perfect anyway.